src/client/app/index.ts

```diff
@@ -6,7 +6,7 @@ import {
   onMounted,
   watch
 } from 'vue'
-import Theme from '/@theme/index'
+import Theme, { VPBadge } from '/@theme/index'
```

Comment:
`@theme/index` is an alias to the theme entry, which could be the user's custom theme. So `{ VPBadge }` is not guaranteed to be exported.